### DISCUSSION TITLE
Announcing an Updated, Accessible Comment Box for Issues, Pull Requests and Discussions

### DISCUSSION THREAD

**Discussion by ghostinhershell** [community/community#73354] [Accessibility]
Hey there! 👋🏾 

We're proud to announce a new, more platform consistent and accessible comment box. You'll see this update when submitting comments for Issues, Pull Requests and Discussions:
![This image is a screenshot of the new comment box](https://github.com/community/community/assets/14021469/5cff8ac7-65e9-4950-8353-25e6e5f2a16a)

This release is a testament to our ongoing efforts to provide an exceptional user experience, making GitHub more intuitive, consistent, and accessible across the platform.

You can learn more by checking out the GitHub Blog here post here: [The Comment Box Becomes More Consistent and Accessible](https://github.blog/changelog/2023-10-31-the-comment-box-becomes-more-consistent-and-accessible/).

Got any feedback to share? Check out the feedback discussion for this update here: [An updated comment box in issues, pull requests and discussions](https://github.com/orgs/community/discussions/73066).

Cheers! 🚀

**Comment by over-light** [comment]
My first answer.

**Comment by over-light** [comment]
Thank you.

**Comment by over-light** [comment]
This release is making GitHub more intuitive, consistent, and accessible across the platform. Thank you.
